daml-lf/encoder/src/main/scala/com/digitalasset/daml/lf/archive/testing/DamlLfEncoder.scala

```diff
@@ -130,8 +130,10 @@ private[digitalasset] object DamlLfEncoder extends App {
 
   val MANIFEST =
```

Comment:
Please add a comment here that the maximum line length for these manifests is 72 characters and to be careful when changing this. We've been bitten by that in the past. https://docs.oracle.com/javase/6/docs/technotes/guides/jar/jar.html#JAR%20Manifest.

Even better would probably be to use a proper encoder for creating the manifest.

Reply:
Note that this is only used for testing so even if we do mess it up the fallout is relatively small.

Reply:
We may use the encoder to test ledger implementation, since it is the only way to generate from HEAD dars containing old daml-lf version that the compiler does not support.

I will create a new PR too tackle @dajmaki remark.